addons/stock/static/src/js/report_stock_forecasted.js

```diff
@@ -123,7 +123,9 @@ const ReplenishReport = clientAction.extend({
             withControlPanel: false,
             context: {fill_temporal: false},
         };
-        const GraphView = viewRegistry.get("graph");
+        const viewArch = new DOMParser().parseFromString(viewInfo.arch, "text/xml");
+        const viewRegistryKey = viewArch.documentElement.getAttribute("js_class");
+        const GraphView = viewRegistry.get(viewRegistryKey || viewInfo.type);
```

Comment:
for the case view.js, it has been modified similarly to what I have written after 15.0 but we can keep your simple version. Just put "graph" instead of viewInfo.type as fallback

Reply:
I see what you mean now, I was looking at `15.0` code.
I've just changed it to be aligned to the new `view.js` code, simlar to your suggestion

Reply:
Thank you for the PR.